kernel/src/main/java/org/kframework/compile/GeneratedTopFormat.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-2018 K Team. All Rights Reserved.
+package org.kframework.compile;
+
+import org.kframework.builtin.KLabels;
+// import org.kframework.builtin.Sorts;
+import org.kframework.definition.Module;
+import org.kframework.definition.NonTerminal;
+import org.kframework.definition.Production;
+import org.kframework.definition.ProductionItem;
+import org.kframework.definition.Sentence;
+import scala.collection.Set;
+import scala.collection.JavaConverters;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.kframework.Collections.*;
+import static org.kframework.definition.Constructors.*;
+
+public class GeneratedTopFormat {
+
+    private static Production resolve(Production prod) {
+        if (prod.klabel().isDefined() && prod.klabel().get().equals(KLabels.GENERATED_TOP_CELL)) {
+            List<Integer> cellPositions = new ArrayList<Integer>();
+            int i = 1;
+            for (ProductionItem p: JavaConverters.seqAsJavaList(prod.items())) {
+                if (p instanceof NonTerminal) {
+                    NonTerminal nt = (NonTerminal) p;
+                    // if (! nt.sort().equals(Sorts.GeneratedCounterSort)) {
+                    if (! nt.sort().name().equals("GeneratedCounterCell")) {
```

Comment:
why was this commented out?

Reply:
It was an issue with how I was access the Scala object.